src/didactWebView.ts

```diff
@@ -266,26 +266,37 @@ export class DidactWebviewPanel {
 		// And the uri we use to load this script in the webview
 		const scriptUri = scriptPathOnDisk.with({ scheme: 'vscode-resource' });
 
-		let cssFile = 'webviewslim.css';
-		if (this.isAsciiDoc) {
-			cssFile = 'asciidoctor.css';
-		}
+		const bulmaCssHtml = `<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.8.0/css/bulma.min.css">`;
+
+		const adCssPathOnDisk = vscode.Uri.file(
+			path.join(this._extensionPath, 'media', 'asciidoctor.css')
+		);
+		const adCssUri = adCssPathOnDisk.with({ scheme: 'vscode-resource' });
+		const adUriHtml = `<link rel="stylesheet" href="${adCssUri}"/>`;
 
 		const cssPathOnDisk = vscode.Uri.file(
-			path.join(this._extensionPath, 'media', cssFile)
+			path.join(this._extensionPath, 'media', 'webviewslim.css')
 		);
 		const cssUri = cssPathOnDisk.with({ scheme: 'vscode-resource' });
+		const cssUriHtml = `<link rel="stylesheet" href="${cssUri}"/>`;
+
+		let stylesheetHtml = '';
+		if (this.isAsciiDoc) {
+			stylesheetHtml = `${adUriHtml}\n ${cssUriHtml}\n`;
+
```

Comment:
useless empty line

Reply:
I think it makes it more readable, so I'm going to leave it in.